test/test_poolmanager.py

```diff
@@ -83,6 +83,11 @@ def test_nohost(self, url):
         with pytest.raises(LocationValueError):
             p.connection_from_url(url=url)
 
+    def test_invalid_host(self):
```

Comment:
Can you rename it to a more appropriate name?

Reply:
Renamed to `test_host_with_invalid_idna_labels`